ANSWERS.md

```diff
@@ -0,0 +1,8 @@
+- [ ] What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.

```

Comment:
PropTypes are used to check the variable type of the props being passed, and logs an error if they are wrong.